week2/homework/bonus/bonus.js

```diff
@@ -0,0 +1,77 @@
+const program = require('commander');
```

Comment:
commander 👍

Reply:
It was very new, I hope I could do it properly.